### ISSUE TITLE
Wrong PYTHONPATH for Python 3.7

### ISSUE THREAD

**Issue by axstet** [haiwen/seafile-server#335]
"python3.6" is hard coded for PYTHONPATH variable in the following files:
reset-admin.sh
seahub.sh
setup-seafile-mysql.sh

Compiling with Python 3.7 requires PYTHONPATH to be adjusted to "python3.7".

**Comment by jobenvil** [comment]
yep, a workaround is a simbolic link to 3.6. I prefer just a "python3" without minor version and work on the system with `update-alternatives`

**Comment by VJean** [comment]
It's also harcoded in seafile-controller.c , in the `setup_python_path()` function.

I also ended up with a symlink.
